src/hooks/useOpenSmallBalances.js

```diff
@@ -1,21 +1,31 @@
-import { useCallback } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { setOpenSmallBalances } from '../redux/openStateSettings';
+import { useCallback, useEffect } from 'react';
+import { useMMKVBoolean } from 'react-native-mmkv';
 
 export default function useOpenSmallBalances() {
-  const dispatch = useDispatch();
-
-  const isSmallBalancesOpen = useSelector(
-    ({ openStateSettings: { openSmallBalances } }) => openSmallBalances
+  const [isSmallBalancesOpen, setIsSmallBalancesOpen] = useMMKVBoolean(
+    'small-balances-open'
```

Comment:
If we do care about persisting this, then can we have it per `accountAddress` as well to match the behavior in other sections

Reply:
sure